wolfssl/wolfcrypt/wc_port.h

```diff
@@ -133,10 +150,7 @@
             #define WIN32_LEAN_AND_MEAN
         #endif
         #if !defined(WOLFSSL_SGX) && !defined(WOLFSSL_NOT_WINDOWS_API)
-            #if defined(_WIN32_WCE) || defined(WIN32_LEAN_AND_MEAN)
-                /* On WinCE winsock2.h must be included before windows.h */
```

Comment:
Is removing this safe for a WinCE build. I see a couple locations where this comment and the include was removed.

Reply:
It is Microsoft hack which disable inclusion of winsock.h at all but mainly from windows.h that winsock2.h can be included anywhere.
It is reason why win CE guard is useless. Inclusion of winsock.h is blocked by _WINSOCKAPI_ macro which control inclusion of winsock.h therefore winsock2.h can be included anywhere not before windows.h.

Reply:
Thanks for the clarification on this. Looks like adding the define of _WINSOCKAPI_ before including windows.h resolves the issue the previous port to WinCE was having.